builder/builder.go

```
@@ -1,49 +1,128 @@
 		return out, err
 	}
 
 	return out, nil
 }
+
+func applyTrustedBuildConfig(assessmentDir string, testDir string) (string, error) {
+	buildConfigDir := filepath.Join(testDir, buildConfigDirName)
+
+	info, err := os.Stat(buildConfigDir)
+	if err != nil || !info.IsDir() {
+		return "", fmt.Errorf("test repo has no %s/; building with the submission's own build config\n", buildConfigDirName)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)